lib/c2_version.rb

```diff
@@ -13,16 +13,12 @@ def attributes
     end
   end
 
+  def live_version
+    item
+  end
+
   def diff
```

Comment:
do we have a unit test for this? if not, should we add one? seems like that would be as valuable as the feature spec. I know this logic is mostly from papertrail and we don't want to test papertrail internals, but there was a logic change here that is important so seems like that should be tested and the unit level somehow?